<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepunereExtragereWindow</class>
 <widget class="QMainWindow" name="DepunereExtragereWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titluLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TITLU</string>
    </property>
   </widget>
   <widget class="QLabel" name="contLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cont</string>
    </property>
   </widget>
   <widget class="QLabel" name="disponibilLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>disponibil</string>
    </property>
   </widget>
   <widget class="QLabel" name="valoareLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Valoare</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valoareEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Accepta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renuntaButon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Renunta</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
